sky/provision/kubernetes/utils.py

```diff
@@ -1310,3 +1354,14 @@ def get_head_pod_name(cluster_name_on_cloud: str):
     # label, but since we know the naming convention, we can directly return the
     # head pod name.
     return f'{cluster_name_on_cloud}-head'
+
+
+def get_autoscaler_type(
+) -> Optional[kubernetes_enums.KubernetesAutoscalerType]:
+    """Returns the autoscaler type by reading from config"""
+    autoscaler_type = skypilot_config.get_nested(['kubernetes', 'autoscaler'],
+                                                 None)
+    if autoscaler_type is not None:
+        autoscaler_type = kubernetes_enums.KubernetesAutoscalerType.from_str(
+            autoscaler_type)
```

Comment:
nit: seems we have the schema check for the autoscaler_type already, should we just do `kubernetes_enums.KubernetesAutoscalerType(autoscaler_type)`?